iris/tests/bookmark/delete_bookmark_from_library.py

```diff
@@ -0,0 +1,59 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from iris.test_case import *
+
+
+class Test(BaseTest):
+
+    def __init__(self):
+        BaseTest.__init__(self)
+        self.meta = 'Delete a bookmark from Library '
+        self.test_case_id = '169267'
+        self.test_suite_id = '2525'
+        self.locales = ['en-US']
+
+    def run(self):
+        soap_wiki_tab_pattern = Pattern('soap_wiki_tab.png')
+        delete_bookmark_pattern = Pattern('delete_bookmark.png')
+
+        navigate(LocalWeb.SOAP_WIKI_TEST_SITE)
```

Comment:
- Replace manual bookmarking with TEN_BOOKMARKS profile
- Rewrite assert comments accordingly

Reply:
Edit space in test case title to: "Edit this bookmark" panel appears after a page is already bookmarked